app/assets/v2/js/cart-data.js

```diff
@@ -9,19 +9,16 @@ class CartData {
   }
 
   static cartContainsGrantWithId(grantId) {
-    grantId = String(grantId);
+    grantId = Number(grantId);
     const cart = this.loadCart();
-    const idList = cart.map(grant => {
-      return grant.grant_id;
-    });
+    const idList = cart.map((grant) => Number(grant.grant_id));
 
     return idList.includes(grantId);
   }
 
   static share_url(title) {
     const donations = this.loadCart();
-    let bulk_add_cart = 'https://gitcoin.co/grants/cart/bulk-add/';
-
+    let bulk_add_cart = `${window.location.host}/grants/cart/bulk-add/`;
```

Comment:
Do we want to change anything about the `bulk-add` endpoint? @octavioamu my understanding was that you wanted to remove it and instead have the Share Cart URL look like `https://gitcoin.co/grants/cart?ids=<grantIdsList>`. However, I realized one issue with that is the preview links that you see on Twitter would be broken, and would no longer say "3 grants in cart shared by mds1"

Reply:
Oh i think we should still retain the functionality given the bulk-add endpoint has a few extra logic like being able to add grants using

- CLR pk
- category pk
- subcategory pk
- ensuring grants owned by you don't get added 

^ This is useful for the Add All to Cart functionality 

let's see what @octavioamu was getting at